rbac/internal/views.py

```diff
@@ -95,6 +97,37 @@ def list_unmodified_tenants(request):
     return HttpResponse(json.dumps(payload), content_type="application/json")
 
 
+def list_tenants(request):
+    """List tenant details.
+
+    GET /_private/api/tenant/?ready=<true|false>&limit=<limit>&offset=<offset>
+    """
+    limit = int(request.GET.get("limit", 0))
```

Comment:
wonder if we should use a positive number as default

Reply:
If we use `0` it will default to not being set, which is what we want, since `if limit` won't pass.

Reply:
oh, got it 👍